### PR TITLE
Add database schema blog

### PR DESCRIPTION
**Pull request by joan-ing**
Based on this post: https://www.notion.so/xata/Designing-effective-database-schemas-ff4d177394cf43468ec8e6a52624ef4d

### CODE REVIEW

File: database-schema-design.mdx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+### Visualizing the database schema
+
+Think of a database schema as a filing system, like a huge cabinet with lots of drawers, each containing different categories of files. In this analogy, the cabinet is the whole database, and each drawer is a table within the database. Each file in the drawer is a [row in the table](https://xata.io/docs/concepts/data-model), and the labels on the files are the columns.
+
+If your filing system is a mess and labeled incorrectly (i.e. your schema is poorly designed) it can be a headache to find what you need and get the data you want from your database. To avoid this, it's important to create a well-organized [schema](https://xata.io/docs/concepts/schema) that can easily store and retrieve data.
+
+<div style="width:100%;height:0;padding-bottom:50%;position:relative;"><iframe src="https://giphy.com/embed/AaBhK3dHsk0XS" width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div><p><a href="https://giphy.com/gifs/look-reports-devs-AaBhK3dHsk0XS">via GIPHY</a></p>
```
Comment (tsg) [inline database-schema-design.mdx:31]: This doesn't seem to work. I think you need to save it as gif, copy it in the images folder, then references it via the:
```
![Image caption](link)
```

syntax. Let me know if you need help with this.

File: database-schema-design.mdx [hunk 2]
```diff
@@ -0,0 +1,208 @@
+---
+title: 'Designing your database schema'
+description: 'Understanding database schema design - a quick guide with examples'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-rename.png
```
Comment (tsg) [inline database-schema-design.mdx:5]: Is `xata-rename.png` correct here?

**Review by tsg — COMMENTED** [review]
There is an issue with the giphy, otherwise LGTM.

**Review by philkra — APPROVED** [review]
LGTM

**Pull request merged**